our-process/policies.md

```diff
@@ -65,41 +68,50 @@ external guest editor will be recruited to lead the package review.
 
 ## After Acceptance: Package Ownership and Maintenance
 
-Authors of contributed packages essentially maintain the same ownership they
-had prior to their package joining the pyOpenSci suite. Package authors will
-continue to maintain
-and develop their software after acceptance into pyOpenSci. Unless explicitly added as
-collaborators, the pyOpenSci team will not interfere with day to day operations.
+Package authors are expected to maintain and develop their software and retain
+ownership of it after acceptance into pyOpenSci, as per the peer review agreement
+acknowledged upon submission. This maintenance commitment should last for at
+least two years. The pyOpenSci team will not interfere with day-to-day tool
+maintenance unless explicitly added as collaborators.
 
 If you need to step down from maintaining your accepted pyOpenSci package, please
-please notify the EiC / software review lead of this as soon as you can. We
-will talk with you about your tool and work together to either find a new maintainer or sunset the tool depending upon what makes the most sense.
+promptly notify the pyOpenSci Editor-in-Chief or Software Review Lead. pyOpenSci
+will collaborate with you to either:
+
+- Find a new maintainer or
+- Archive the tool, depending on what best suits your specific scientific Python
+  package.
 
-We will reach out to our package maintainers each year to check in on how
-maintenance is going.
+We will reach out to our package maintainers each year to assess the status of
+maintenance and to see if there are any updates we can highlight through our social channels.
 
-### Maintainer Responsiveness
-If package maintainers do not respond in a timely manner to requests for
-package fixes, we will remind the maintainer a number of times. After 3 months
-(or shorter time frame, depending on how critical the fix is) we will discuss
-the future of the package as a part of our pyOpenSci ecosystem.
+### Maintenance Tracking
 
-* If a package becomes completely un-maintained we will highlight that fact and
-remove it as a vetted tool in our ecosystem.
-* If a sub-community decides to fork and maintain that package we are open to working with the new maintainers to keep the package within our ecosystem.
+pyOpenSci is building a system to track package metrics, including issues, pull requests, and dates of
+the last release and commits to the package repository. Based on activity in
+the repository:
 
-<!-- Should authors abandon the maintenance of an actively used package in our suite,
-we will consider petitioning PyPI to transfer package maintainer status to pyOpenSci. -->
+- Packages not updated within 6 months will receive a flag on the website in the form of a small icon.
```

Comment:
- Packages with low activity (no new commits or releases) in the past six months will be noted by a small icon flag on the website.

Reply:
Do you have an idea what this icon would be? As @hamogu said, if a package is ultra-stable, 6-month period might be unfair for flaggin this package, especially if the icon can be something alarming like a ❌ .

Reply:
ok i hear you all. we dont' want to alarm. i think (but let's talk about this). 

1. a 6 month flag - it can even be a small green flag that's a different shade of green that just says - hey this package hasn't been updated in 6 month (updated meaning no new commits, releases or issue activity??) i know issue activity could be from outside users so perhaps that is not a good marker
2. 12 month flag - maybe the status goes to yellow. that is when we'd directly reach out to maintainers. 

thoughts on something like that?